src/utils/mask.ts

```diff
@@ -21,6 +35,46 @@ function masker(value: string, pattern: string, options: any) {
   return toPattern(value, { pattern, ...options });
 }
 
+/**
+ * function masker(
+ * @param {string} value
+ * @param {any} options
+ * @returns {string}
+ */
+function currencyMasker(value: string = "0", options: any) {
+  const { 
+    prefix,
+    decimalSeparator,
+    groupSeparator,
+    precision,
+    groupSize,
+    secondaryGroupSize,
+    fractionGroupSeparator,
+    fractionGroupSize,
+    suffix
+  } = options;
+
+  const precisionDivider = parseInt(1 + "0".repeat(precision ||0))
+  const number = parseInt(value) / precisionDivider;
+
+  const formatter = {
+    prefix,
+    decimalSeparator,
+    groupSeparator,
+    groupSize: groupSize || 3,
```

Comment:
On later versions, would be nice to put in the docs the specification "if no group size is passed, then the default resolves to 3". No magical behaviors

Reply:
Did this last commit thanks for suggestion